DEVELOPMENT.md

```diff
@@ -0,0 +1,53 @@
+# Development
+This document will help to setup your development environment and running tests for milvus java sdk. If you encounter a problem, please file an issue.
+
+## Getting started
+
+### Prerequisites
+    -   Java 8 or higher
+    -   Apache Maven
+
+## Building Milvus java SDK
+call the following command to generate protobuf related code
+```shell
+  mvn install
+```
+
+## Building Milvus
+see detailed information at:
+https://github.com/milvus-io/milvus/blob/master/DEVELOPMENT.md
+
+## Start a Milvus cluster
+You need to start a latest milvus cluster to test the java SDK, see instructions at:
+https://milvus.io/docs/v2.0.0/install_standalone-docker.md
+
+### Unit Tests
+All unit test is under director src/test, TBD
+
+## GitHub Flow
+Milvus SDK repo follows the same git work flow as milvus main repo, see
+https://milvus.io/community/contributing_to_milvus.md
+
+Some basic git instruction
+```shell
+How to fork a repo
+   https://docs.github.com/cn/get-started/quickstart/fork-a-repo
+   
+Clone mivlus sdk repo
+    git clone git@github.com:milvus-io/milvus-sdk-java.git
+   
+Add personal remote repo 
+    git remote add devrepo myrepoaddress
+    
+Fetch remote repo
+    git fetch devrepo
+    
+Checkou on devrepo master branch to local as localdev
+    git checkout -b localdev devrepo/master
+    
+Push to dev repo master branch
+    git push devrepo localdev:master
```

Comment:
Should checkout a feature branch from origin/master, and push this feature branch to the devrepo, create a pull request from the feature branch.